sale_variant_configurator/models/sale_order.py

```diff
@@ -31,6 +31,11 @@ class SaleOrderLine(models.Model):
         string="Product Template (no related)",
     )
     product_id = fields.Many2one(required=False)
+    product_uom_category_id = fields.Many2one(
+        comodel_name="uom.category",
+        compute="_compute_product_uom_category_id",
+        related=False,
```

Comment:
Why are you removing this as related? If it's because the related is not "acting on time", please try to look for another option, as this can have side effects.

Reply:
We define `related=False` to avoid it being a related (and therefore a compute). Another possible alternative option is to create another `product_tmpl_uom_category_id` field and modify the domain of the `product_uom` field, what do you prefer?

Reply:
OK, now I see the problem seeing https://github.com/OCA/OCB/blob/6c40bebb4eb0ca9566378ac7b82b50f9c1fbb0bc/addons/sale/models/sale_order_line.py#L94 when no `product_id` is still defined.

Reply:
Added comment to clarify.